src/components/Sidebar/sidebar-props.ts

```diff
@@ -0,0 +1,11 @@
+import { EXPAND_MENU } from './expand-menu-enum';
```

Comment:
As it isn't used by more than one component, this doesn't need to be in its own file, can just be placed inline in Sidebar.

Reply:
This is also used by SidebarLinkItem, SidebarLinkMenu and left-sidebar.js as an enum now, replacing the frozen object 'enum' used before, should I still move it?

Reply:
Ah I understand you mean the whole file now after reading the other comment, got it!